scripts/pi-hole/js/messages.js

```diff
@@ -73,6 +73,9 @@ function renderMessage(data, type, row) {
         "&uarr;</pre>"
       );
 
+    case "DNSMASQ_CONFIG":
+      return "FTL failed to start due to " + row.message;
```

Comment:
Just to be safe, do we need to escape this?

Reply:
The message can only contain text and file paths so the most exotic thing we can find in here is `/`